conf/celeryconfig.py

```diff
@@ -5,13 +5,12 @@
 from celery.schedules import crontab
 
 from chronam.settings import *  # NOQA
-from chronam.settings import INSTALLED_APPS
 
 APP_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 
 
-CELERY_IMPORTS = ("chronam.core.tasks",)
-CELERYD_LOG_FILE = os.path.join("/logs/celery", "celery.log")
+CELERY_IMPORTS = ("chronam.core.tasks")
```

Comment:
This changes the meaning — I think this change should be reverted: 

```python
>>> ("chronam.core.tasks",)
('chronam.core.tasks',)
>>> ("chronam.core.tasks")
'chronam.core.tasks'
```

Reply:
I didn't think the original code was problem but was just following your suggestion